root/static/scripts/common/components/TagEditor.js

```diff
@@ -623,9 +624,10 @@ function createInitialTagState(
   });
 
   // Always show upvoted user tags (affects sidebar)
-  for (const t of userTagsByName.values()) {
-    if (t.vote > 0 && !used.has(t.tag)) {
-      combined.push(t);
+  for (const tagName of Object.keys(userTagsByName)) {
+    const tag = userTagsByName[tagName];
+    if (tag.vote > 0 && !used.has(tagName)) {
+      combined.push(tag);
```

Comment:
`userTagsByName` is a plain object rather than a `Map` now, so it doesn't have a `.values()` method. I've changed it to loop over the object keys instead.